OpenOversight/migrations/env.py

```diff
@@ -19,6 +14,12 @@
 logger = logging.getLogger("alembic.env")
 
 
+config.set_main_option(
+    "sqlalchemy.url", current_app.config.get("SQLALCHEMY_DATABASE_URI")
```

Comment:
Could you change this to: `"sqlalchemy.url", current_app.config.get(KEY_DATABASE_URI)`?